mail-filter/spamassassin/spamassassin-4.0.0.ebuild

```diff
@@ -0,0 +1,321 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-functions systemd toolchain-funcs
+
+MY_P="Mail-SpamAssassin-${PV//_/-}"
+S="${WORKDIR}/${MY_P}"
```

Comment:
```S``` goes after ```SRC_URI```. See skel.ebuild.

Reply:
Done.

Reply:
You may also want to push update, because it is still here

Reply:
Yeah, needed another push.  Thanks.